lib/src/cmaf/ham/mapper/dash/mapDashToHam.ts

```diff
@@ -43,13 +44,19 @@ function mapTracks(
 		throw new Error('Error: AdaptationSet is undefined');
 	}
 	const type = getContentType(adaptationSet, representation);
+	const frameRate = getFrameRate(adaptationSet, representation).split('/');
+	const frameRateNum = parseInt(frameRate.at(0) ?? '');
```

Comment:
Maybe zero and one should be added as a constant, something like NUMERATOR_POSITION = 0 and DENOMINATOR_POSITION = 1

Reply:
I agree, maybe just the word without `position` could work too.

Reply:
Done!